.github/workflows/update_wheels.yml

```diff
@@ -0,0 +1,42 @@
+name: Release
+
+on:
+  workflow_dispatch:
+  pull_request:
```

Comment:
```suggestion
```
I guess these shouldn't be needed. Other than that LGTM, guess we have to actually make a release to test.